crates/common/tedge_config/src/tedge_toml/tedge_config.rs

```diff
@@ -226,6 +227,10 @@ define_tedge_config! {
         #[tedge_config(reader(private))]
         url: ConnectUrl,
 
+        /// Cumulocity tenant ID
+        #[tedge_config(example = "t12345678")]
+        tenant_id: String,
+
```

Comment:
It the `tenant_id` still useful?

Reply:
That was the first line that I had removed, but it accidentally got stashed away. Resolved again in 7a827c8